cmdline/maven/tests/fail-on-warnings2/golden

```diff
@@ -0,0 +1,12 @@
+${basedir}/src/main/java/test/App.java:24: warning: [Convert_to_Lambda_or_Member_Reference] This anonymous inner class creation can be turned into a lambda expression.
```

Comment:
Does this work on Windows? With `\` instead of `/`?

Reply:
I don't think it is possible to build Jackpot outside of Linux (and maybe Mac). I am personally fine with that. If someone wants to add this ability, PRs are welcome!